lib/games/sendUserNotification.js

```diff
@@ -0,0 +1,69 @@
+const http = require('../util/http.js').func
+
+exports.required = ['universeId', 'userId', 'assetId', 'parameters']
+exports.optional = ['jar']
+
+// Docs
+/**
+ * ☁️ Send a universe notification to a user.
+ * @category Game
+ * @alias sendUserNotification
+ * @param {number} universeId - The id of the universe.
+ * @param {number} userId - The id of the target player.
+ * @param {string} assetId - The asset id of the notification.
+ * @param {UserNotificationPayloadParameters} parameters - The notification parameters.
+ * @returns {Promise<boolean>}
+ * @example const noblox = require("noblox.js")
+ * // Set API key
+ * const parameters = {
+ *      myMessage: { stringValue: "Hello world!" }
+ * }
+ * const assetId = "774d62e5-3414-b84a-89dd-77d96f1a3d33"
+ * noblox.sendUserNotification(4434277335, 1210019099, assetId, parameters)
+ **/
+
+function sendUserNotification (universeId, userId, assetId, parameters, jar) {
+  return new Promise((resolve, reject) => {
+    const httpOpt = {
+      url: `//apis.roblox.com/cloud/v2/users/${userId}/notifications`,
+      options: {
+        json: true,
+        resolveWithFullResponse: true,
+        jar,
+        method: 'POST',
+        body: {
+          source: {
+            universe: `universes/${universeId}`
+          },
+          payload: {
+            type: 'MOMENT',
+            messageId: assetId,
+            parameters
+          }
+        },
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      }
+    }
+
+    return http(httpOpt)
+      .then(function (res) {
+        if (res.statusCode === 200) {
+          resolve(true)
+        } else {
+          if (typeof res.body === 'string') {
+            reject(new Error(`[${res.statusCode}] ${res.statusMessage} ${res.body}`))
+          } else {
+            const data = Object.assign(res.body)
+            reject(new Error(`[${res.statusCode}] ${data.Error} ${data.Message}`))
```

Comment:
The error handling doesn't really work here.

V2 open cloud doesn't use the error property as far as I know (and Roblox generally follows camel case for this)

Reply:
It does catch, so it's not completely broken.

But Roblox returns a v2 error model because it's a v2 endpoint, so it comes out as `[code] undefined undefined`

The properties would be `.code` and `.message`

Reply:
I never got that issue with it. Have you tested it?